katago-analyze-sgf-daemon.js

```diff
@@ -124,12 +125,23 @@ class Engine extends EventEmitter {
     if (this.katago && !this.katago.killed) {
       return
     }
-    this.katago = spawn(this.katagoPath, [
-      'analysis',
-      '-config',
-      this.analysisConfig,
-      '-quit-without-waiting',
+    if (this.modelPath == null) {
+      this.katago = spawn(this.katagoPath, [
+        'analysis',
+        '-config',
+        this.analysisConfig,
+        '-quit-without-waiting',
+      ])
+    } else {
+      this.katago = spawn(this.katagoPath, [
+	'analysis',
+	'-config',
+	this.analysisConfig,
+	'-model',
+	this.modelPath,
+	'-quit-without-waiting',
```

Comment:
Instead of having different calls to `spawn` in the two branches of the `if` statement, just construct the arguments to `spawn` correctly and then call `spawn` once.